libowfat: 0.31 -> 0.32

r-ryantm opened the pull request:
Semi-automatic update generated by https://github.com/ryantm/nixpkgs-update tools. This update was made based on information from https://repology.org/metapackage/libowfat/versions.



meta.description for libowfat is: '""'.


<details>
<summary>
Checks done (click to expand)
</summary>

- built on NixOS

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- directory tree listing: https://gist.github.com/ed47edb4c0c76cd69667131b796a4c33
- du listing: https://gist.github.com/b1840c049cdb43aaf1dffa81ba8557ae

</details>
<details>
<summary>
Instructions to test this update (click to expand)
</summary>

One-time optional setup to skip building using Cachix:
```
nix-shell -p cachix --run 'cachix use r-ryantm'
```

Test this update by entering a nix shell, seeing what is inside the
result, and if applicable, running some binaries:
```
nix-shell --pure -I nixpkgs=https://github.com/r-ryantm/nixpkgs/archive/796fe1d56ed27d4588757341d295d25f197eef42.tar.gz -p libowfat
ls -la /nix/store/rbq5l7nrglwfc61wb0x4684hw823hd4b-libowfat-0.32
```

</details>
<br/>

r-ryantm committed:
libowfat: 0.31 -> 0.32

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/libowfat/versions

GrahamcOfBorg commented:
<!--REQUEST_ID=4c03b886-d983-497c-b4bc-fc9b0c392749-->
No attempt on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48414&attempt_id=e200b986-95ce-4234-90df-94ce2071ec3e)

The following builds were skipped because they don't evaluate on x86_64-darwin: libowfat

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowUnsupportedSystem = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowUnsupportedSystem = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=4c03b886-d983-497c-b4bc-fc9b0c392749-->
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48414&attempt_id=d4847430-8655-4d3b-890a-78e2af519ab3)

Attempted: libowfat

<details><summary>Partial log (click to expand)</summary><p>

```
install -m 644 libowfat.a /nix/store/w1p0kxr7qd12palk3f31634f4i9v4ppk-libowfat-0.32/lib
post-installation fixup
moving /nix/store/w1p0kxr7qd12palk3f31634f4i9v4ppk-libowfat-0.32/man to /nix/store/w1p0kxr7qd12palk3f31634f4i9v4ppk-libowfat-0.32/share/man
shrinking RPATHs of ELF executables and libraries in /nix/store/w1p0kxr7qd12palk3f31634f4i9v4ppk-libowfat-0.32
gzipping man pages under /nix/store/w1p0kxr7qd12palk3f31634f4i9v4ppk-libowfat-0.32/share/man/
strip is /nix/store/p9akxn2sfy4wkhqdqa3li97pc6jaz3r1-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/w1p0kxr7qd12palk3f31634f4i9v4ppk-libowfat-0.32/lib
patching script interpreter paths in /nix/store/w1p0kxr7qd12palk3f31634f4i9v4ppk-libowfat-0.32
checking for references to /build in /nix/store/w1p0kxr7qd12palk3f31634f4i9v4ppk-libowfat-0.32...
/nix/store/w1p0kxr7qd12palk3f31634f4i9v4ppk-libowfat-0.32
```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=4c03b886-d983-497c-b4bc-fc9b0c392749-->
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48414&attempt_id=1d75bddc-97c9-4358-9177-3300271f569a)

Attempted: libowfat

<details><summary>Partial log (click to expand)</summary><p>

```
install -m 644 libowfat.a /nix/store/rbq5l7nrglwfc61wb0x4684hw823hd4b-libowfat-0.32/lib
post-installation fixup
moving /nix/store/rbq5l7nrglwfc61wb0x4684hw823hd4b-libowfat-0.32/man to /nix/store/rbq5l7nrglwfc61wb0x4684hw823hd4b-libowfat-0.32/share/man
shrinking RPATHs of ELF executables and libraries in /nix/store/rbq5l7nrglwfc61wb0x4684hw823hd4b-libowfat-0.32
gzipping man pages under /nix/store/rbq5l7nrglwfc61wb0x4684hw823hd4b-libowfat-0.32/share/man/
strip is /nix/store/vcc4svb8gy29g4pam2zja6llkbcwsyiq-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/rbq5l7nrglwfc61wb0x4684hw823hd4b-libowfat-0.32/lib
patching script interpreter paths in /nix/store/rbq5l7nrglwfc61wb0x4684hw823hd4b-libowfat-0.32
checking for references to /build in /nix/store/rbq5l7nrglwfc61wb0x4684hw823hd4b-libowfat-0.32...
/nix/store/rbq5l7nrglwfc61wb0x4684hw823hd4b-libowfat-0.32
```
</p></details>

Mic92 commented:
It looks like this update break `opentracker`.

xeji commented:
@GrahamcOfBorg build gatling opentracker

GrahamcOfBorg commented:
<!--REQUEST_ID=d679d454-01a6-40c0-abbf-37a3113f0043-->
No attempt on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48414&attempt_id=ecab5d91-e329-444f-9083-b32d55cd00da)

The following builds were skipped because they don't evaluate on x86_64-darwin: gatling, opentracker

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowUnsupportedSystem = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowUnsupportedSystem = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=d679d454-01a6-40c0-abbf-37a3113f0043-->
Failure on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48414&attempt_id=9bd2ea6e-edb4-44a9-b231-4b84456a8c84)

Attempted: gatling, opentracker

<details><summary>Partial log (click to expand)</summary><p>

```
rm -f tryalloca.o
gcc -c gatling.c -o gatling.o -I. -pipe -Wall -O2 -fomit-frame-pointer -I/usr/local/include -DUSE_ZLIB
In file included from gatling.c:2:0:
gatling.h:7:10: fatal error: io.h: No such file or directory
 #include "io.h"
          ^~~~~~
compilation terminated.
make: *** [GNUmakefile:116: gatling.o] Error 1
builder for '/nix/store/7h46hg2yf9fmfl1958d27ahw0p00108k-gatling-0.15.drv' failed with exit code 2
error: build of '/nix/store/7h46hg2yf9fmfl1958d27ahw0p00108k-gatling-0.15.drv', '/nix/store/cs2yhl74m9myiacnbdw0d7ciax5a0d57-opentracker-2018-05-26.drv' failed
```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=d679d454-01a6-40c0-abbf-37a3113f0043-->
Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48414&attempt_id=22088266-f3c0-4e18-aaf0-6c0c9a20d132)

Attempted: gatling, opentracker

<details><summary>Partial log (click to expand)</summary><p>

```
rm -f tryalloca.o
gcc -c gatling.c -o gatling.o -I. -pipe -Wall -O2 -fomit-frame-pointer -I/usr/local/include -DUSE_ZLIB
In file included from gatling.c:2:0:
gatling.h:7:10: fatal error: io.h: No such file or directory
 #include "io.h"
          ^~~~~~
compilation terminated.
make: *** [GNUmakefile:116: gatling.o] Error 1
builder for '/nix/store/rb2rymklw5wls785apxwdassnswv1nv7-gatling-0.15.drv' failed with exit code 2
error: build of '/nix/store/lng3rwdhyx4y0yh0qhnsd9ig0vvnf1nw-opentracker-2018-05-26.drv', '/nix/store/rb2rymklw5wls785apxwdassnswv1nv7-gatling-0.15.drv' failed
```
</p></details>

xeji reviewed (changes requested):
This breaks both `gatling` and `opentracker`. Looks like some header files are missing.

c0bw3b committed:
libowfat: keep headers under include directory

0.32 wanted to move them under include/libowfat/ 
but it breaks gatling and opentracker

c0bw3b commented:
@GrahamcOfBorg build libowfat gatling opentracker

GrahamcOfBorg commented:
<!--REQUEST_ID=4ae2d142-b051-4dca-8eb2-e8558b2daeeb-->
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48414&attempt_id=ff1eff56-3158-4e74-9e6b-88414fd1a227)

Attempted: libowfat, gatling, opentracker

<details><summary>Partial log (click to expand)</summary><p>

```
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/nxw9n0i3m54hw649445kivhk25h6d62k-opentracker-2018-05-26
shrinking /nix/store/nxw9n0i3m54hw649445kivhk25h6d62k-opentracker-2018-05-26/bin/opentracker
strip is /nix/store/vcc4svb8gy29g4pam2zja6llkbcwsyiq-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/nxw9n0i3m54hw649445kivhk25h6d62k-opentracker-2018-05-26/bin
patching script interpreter paths in /nix/store/nxw9n0i3m54hw649445kivhk25h6d62k-opentracker-2018-05-26
checking for references to /build in /nix/store/nxw9n0i3m54hw649445kivhk25h6d62k-opentracker-2018-05-26...
/nix/store/a0bk2jm77xkimay6p1wjdpba5aai91q6-libowfat-0.32
/nix/store/0i8qbpbgwd0v5msa71aldyip18sdcvjz-gatling-0.15
/nix/store/nxw9n0i3m54hw649445kivhk25h6d62k-opentracker-2018-05-26
```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=4ae2d142-b051-4dca-8eb2-e8558b2daeeb-->
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48414&attempt_id=eb8f9757-b824-4e88-9134-575a51d470b7)

Attempted: libowfat, gatling, opentracker

<details><summary>Partial log (click to expand)</summary><p>

```
shrinking /nix/store/g247mj534wbwc894qhicazd7kqngamc8-gatling-0.15/bin/dl
shrinking /nix/store/g247mj534wbwc894qhicazd7kqngamc8-gatling-0.15/bin/gatling
gzipping man pages under /nix/store/g247mj534wbwc894qhicazd7kqngamc8-gatling-0.15/share/man/
strip is /nix/store/p9akxn2sfy4wkhqdqa3li97pc6jaz3r1-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/g247mj534wbwc894qhicazd7kqngamc8-gatling-0.15/bin
patching script interpreter paths in /nix/store/g247mj534wbwc894qhicazd7kqngamc8-gatling-0.15
checking for references to /build in /nix/store/g247mj534wbwc894qhicazd7kqngamc8-gatling-0.15...
/nix/store/l1hfgm900xmgrlxp9qjiars39vni4ri8-libowfat-0.32
/nix/store/g247mj534wbwc894qhicazd7kqngamc8-gatling-0.15
/nix/store/6l6481naszv23xy6dyy6hl4a9nxm5bi1-opentracker-2018-05-26
```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=68becf9e-a85b-4872-8dd1-f54f19b53c3a-->
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48414&attempt_id=9db5f535-8025-4b47-a1c5-a021fed8f4ef)

Attempted: libowfat

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/l1hfgm900xmgrlxp9qjiars39vni4ri8-libowfat-0.32
```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=68becf9e-a85b-4872-8dd1-f54f19b53c3a-->
No attempt on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48414&attempt_id=a2634195-6ac6-4e4e-b0d2-625461cd0f83)

The following builds were skipped because they don't evaluate on x86_64-darwin: libowfat

<details><summary>Partial log (click to expand)</summary><p>

```

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowUnsupportedSystem = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowUnsupportedSystem = true; }
to ~/.config/nixpkgs/config.nix.


```
</p></details>

GrahamcOfBorg commented:
<!--REQUEST_ID=68becf9e-a85b-4872-8dd1-f54f19b53c3a-->
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48414&attempt_id=42d6b5be-8f05-4994-9597-f03bad2ef401)

Attempted: libowfat

<details><summary>Partial log (click to expand)</summary><p>

```
install -m 644 libowfat.a /nix/store/a0bk2jm77xkimay6p1wjdpba5aai91q6-libowfat-0.32/lib
post-installation fixup
moving /nix/store/a0bk2jm77xkimay6p1wjdpba5aai91q6-libowfat-0.32/man to /nix/store/a0bk2jm77xkimay6p1wjdpba5aai91q6-libowfat-0.32/share/man
shrinking RPATHs of ELF executables and libraries in /nix/store/a0bk2jm77xkimay6p1wjdpba5aai91q6-libowfat-0.32
gzipping man pages under /nix/store/a0bk2jm77xkimay6p1wjdpba5aai91q6-libowfat-0.32/share/man/
strip is /nix/store/vcc4svb8gy29g4pam2zja6llkbcwsyiq-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/a0bk2jm77xkimay6p1wjdpba5aai91q6-libowfat-0.32/lib
patching script interpreter paths in /nix/store/a0bk2jm77xkimay6p1wjdpba5aai91q6-libowfat-0.32
checking for references to /build in /nix/store/a0bk2jm77xkimay6p1wjdpba5aai91q6-libowfat-0.32...
/nix/store/a0bk2jm77xkimay6p1wjdpba5aai91q6-libowfat-0.32
```
</p></details>

c0bw3b reviewed (approved):
Fixed the headers location
